### DISCUSSION TITLE
The book and the tutorial

### DISCUSSION THREAD

**Discussion by katsujukou** [FStarLang/FStar#3233] [Q&A] [answered]
Hi. I'm a complete newbie to F* and I'm working on the book _Proof-Oriented Programming in F*_. 
While reading, I found that the documentation site has a link to [another tutorial](https://fstar-lang.org/tutorial/tutorial.html). 
What is the relationship between these two documents? Should I have started with the tutorial instead of the book?

**Comment by aseemr** [comment, marked as answer]
Sorry for the confusion. The "another tutorial" that you have linked is an older version of the F* tutorial; we should delete it. The book on Proof-Oriented Programming in F* is the one that you should follow.

**Reply by katsujukou** [reply]
Sorry for my late reply.

Thank you for your comment. Now I can proceed confidently in the right direction without any concerns!

**Comment by mtzguido** [comment]
Thanks for the report @katsujukou !

I removed the old tutorial from that URL. Can you say where you found a link to it? We should probably update that pointer.

The old tutorial still remains here, with a note about it being outdated: https://fstar-lang.org/tutorial/old/tutorial.html. This is just for archival purposes, the book is the way to go.

Also, the old tutorial is still under `doc/tutorial` in the F* repo. Should it be...?

**Reply by katsujukou** [reply]
I'm sorry, but I do not remember clearly...
While reading, I was searching for additional learning resources, and I glanced at the contents under the `doc/` directory in the repository. Then I found the document titled _Verified programming in F*_ and I googled that title. I think that served as a pointer for me, maybe...
